server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/SignOffAuditRound.java

```diff
@@ -184,19 +184,9 @@ public String endpointBody(final Request request,
         // We're done!
         cdb.endRound();
 
-        final AuditBoardDashboardASM asm = ASMUtilities.asmFor(
-            AuditBoardDashboardASM.class,
-            String.valueOf(cdb.id()));
-
-        if (null != asm && asm.currentState() == ROUND_IN_PROGRESS) {
-          ASMUtilities.step(ROUND_COMPLETE_EVENT,
-                            AuditBoardDashboardASM.class,
-                            String.valueOf(cdb.id()));
-        }
```

Comment:
if nothing else comes of this, thank you for taking out my trash :recycle:

Reply:
I couldn't get anything to finish taking this bit out, so it's back.